### DISCUSSION TITLE
Webhook trigger based on time of a day [May 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2166] [Discord Threads]
> 🙋‍♂️ **AK**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Webhook trigger like you figured out is only based on record insert/delete/update currently. If you can trigger update for a special "student", then the discussion in this thread will help to configure rest of the details   
####    
#### https://discord.com/channels/661905455894888490/973458332796985376  
#  
> 🙋‍♂️ **AK**  
#### I tried. It's not working well but anyway I can see it only sending 1 row while I need multiple rows..  
#  
> 🙋‍♂️ **AK**  
#### For testing I didn't put any condition.  
#  
> 🙋‍♂️ **AK**  
#### So it finally working without errors (expect at the test webhook function). But I need it to send **all** the rows where the formula there is equals to 1 and it's sending only the updated one if equals to 1...  
#  
> 🙋‍♂️ **AK**  
#### Ok I found myself a solution: using the rest API.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974123991797207100)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
